flask_melodramatiq/lazy_broker.py

```diff
@@ -210,7 +210,7 @@ def __get_secondary_options(self, app):
         options = {
             k[len(prefix):].lower(): v
             for k, v in app.config.items()
-            if k.isupper() and k.startswith(prefix)
+            if k.isupper() and k.startswith(prefix) and v is not None
```

Comment:
This can be problematic when passing `None` as a value to the argument is not the same as not passing the argument at all. I suggest to define a class-value:

```Python
class missing:
    """A missing configuration value"""
```
and then check `v is not missing`. Of course then you would have to pass `DRAMATIQ_BROKER_URL=flask_melodramatiq.missing` in the config.

Reply:
Updated !

Reply:
Great! I will merge it.